web/src/components/new/Navigation.vue

```diff
@@ -51,9 +51,11 @@
             }"
           >
             <v-list-item-content>
-              <v-list-item-title>{{
-                tag.Name.substr(tagType.key.length + 1)
-              }}</v-list-item-title>
+              <v-list-item-title>
+                <v-chip :color="tag.Color">{{
```

Comment:
i liked the optics from before it was a chip, wdyt about putting it as background color into the v-hover?

        <v-hover
          #default="{ hover }"
          :key="tag.Name"
          :style="{ backgroundColor: tag.Color }"
        >